khal/cli.py

```diff
@@ -482,7 +484,7 @@ def isatty(_file):
     @multi_calendar_option
     @click.pass_context
     def interactive(ctx, include_calendar, exclude_calendar):
-        '''Interactive UI. Also launchable via `ikhal`.'''
+        '''Interactive UI. Also launchable via `khal interactive`.'''
```

Comment:
_"Also launchable via `khal interactive`"_ doesn't sound right as a description for `khal interactive`. This is currently shown as the output for `khal interactive --help`:

```console
> khal interactive --help
Usage: khal interactive [OPTIONS]

  Interactive UI. Also launchable via `ikhal`.
```

Reply:
true, removed.